Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -94,11 +94,27 @@ def putMaskOnVTKGrid(data, grid, actorColor=None, cellData=True, deep=True):
             grid.GetPointData().AddArray(ghost)
     return mapper
 
+def handleProjectionEdgeCases(projName, data):
+    # For mercator projection, latitude values of -90 or 90
+    # transformation result in infinity values. We chose -85, 85
```

Comment:
On this, I would like VCS not to scale the viewport but that's something I need to bring up in some meeting. The idea is to fill the screen but to me anytime you use a projection, it really comes out distorted.